templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>Mixing file names, instructions, and descriptions in this paragraph makes it extremely challenging to figure out what is going on. There's also extraneous information in here, like the description of the keys of the JSON file, which the user doesn't even know exists. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>@ngreenwald as a follow up question, for the notebook documentation and slides, how necessary is it to go into detail about the method to select, name, and save the corner FOVs? Should we provide more handholding in this regard or assume a user of Mibitracker should know how to do this?</p><p> </p><p>If the latter, then this documentation can be simplified to a brief 2-3 sentence description of what <code>region_corners_path</code> is, how to name it, and where to place it.</p>

Reply:
<p>Anyone using the notebook will know how to use the commercial instrument. So you won't need to tell them how to name an FOV, but you will need to tell what it needs to be named, etc. </p>

Reply:
Ok cool, @HPiyadasa we can revisit adding separate documentation for this at another time, you could open an issue for it.